garuda-cluster/hourly.txt

```diff
@@ -540,12 +540,15 @@ sibs
 # pulseeffects-git
 librnnoise
 
+# gdm-plymouth
+plymouth
```

Comment:
@RustemB Hi, these two are better to add to Kitsuna's cluster and under "# Issue NNN" so we keep track of who/when it was requested.